# Material and Methods {-}
\chaptermark{Material and Methods}

```{r pckgs, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(kableExtra)
soilparam <- #load data
      read_delim("data/soil.csv",",",)

trt <- data.frame(
   Item = c("Amount,\nForm","Date\n(DAS)","Active\ningredient"),
   Herbicide = c("1 l/ha\nViper", "26","Diflufenican,\nFlorasulam,\nPenoxsulam"),
   Nitrogen = c("40,80\nkg/ha CAN","179,\n201", "Ammonium,\nNitrate,\nCalcium"),
   Magnesium = c("27 kg/ha\nMgO", "179", "Magnesium"),
   Sulfur = c("22 kg/h\nS", "179", "Sulfur"),
   Fungicides = c("1.5 l/ha\nChampion,\n1,0 l/ha\nSeguris", "210,\n232", "Epoxiconazol,\nBoscalid,\nIsopyrazam"),
   Regulators = c(" 0.3 l/ha\nCalma,\n0.5 l/ha\nCCC", "203", "Trinexapac\n-ethyl,\nChlormequat-\nchlorid")
)

# formatting 
   trt$Item <- linebreak(trt$Item, align ="l")
   trt$Herbicide <- linebreak(trt$Herbicide, align ="l")
   trt$Nitrogen <- linebreak(trt$Nitrogen, align ="l")
   trt$Magnesium <- linebreak(trt$Magnesium, align ="l")
   trt$Sulfur <-linebreak(trt$Sulfur, align ="l")
   trt$Fungicides <-linebreak(trt$Fungicides, align ="l")
   trt$Regulators <-linebreak(trt$Regulators, align ="l")
   
   
smpl <- data.frame(
 Name = c("Soil", "Rhizosphere"),
 Source= c("Soil loosely \nbound to root system", "Soil closely \nbound to roots"),
 Method.of.collection = c("After crop was dug out, the\n loose bulk soil was removed, and the\n remaining soil shaken off.", "After “Soil” sample were collected,\n the soil bonded to the root was washed off\nand pelleted.")
)

# formatting

   smpl$Name = linebreak(smpl$Name, align ="l")
   smpl$Source = linebreak(smpl$Source, align ="l")
   smpl$Method.of.collection = linebreak(smpl$Method.of.collection, align ="l")
   
   
prmr <- data.frame(
    target=c("16s rRNA", "amoA", "amoA", "Commamox"),
    kingdom=c("Bacteria and\nArchaea",  "Archaea", "Bacteria", "Bacteria"),
    sequence=c("cctacgggnggcwgcag (341F)/\nattaccgcggctgctgg\n(518R)",
               "atggtctggctwagacg(CrenamoA23F)/\ngccatccatctgtatgtcca\n(CrenamoA616R)",
               "ggggtttctactggtggt(1F)/\ncccctckgsaaagccttcttc(2R)", 
               "aggngaytgggayttctgg(comamoA F)/\ncggacawabrtgaabcccat-(comamoA R)"),
    subtarget = c("V3-Region", "α-subunit", "α-subunit", "Conserved region")
    #name=c("CrenamoA23F/\nCrenamoA616R", "1F/\n2R", "comamoA F/\nR"),
)
#formatting

   prmr$target <- linebreak(prmr$target, align="l")
   prmr$kingdom <- linebreak(prmr$kingdom, align="l")
   prmr$sequence <- linebreak(prmr$sequence, align="l")
```


<!-- Required to number equations in HTML files -->
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>

This study determined how soil-living microorganisms are affected by inputs used in a conventional wheat cropping system. An agricultural field trial with wheat crops treated with conventionally used fertilizers, fungicides and growth regulators in Germany in 2019-2020 was performed. The treated crops were contrasted to control crops without the inputs. From the rhizosphere of the wheat crops samples were taken.   The genomic DNA in the samples was extracted and analyzed with quantitative polymerase chain reaction for their gene copy numbers of the targeted genes. Of interest were genes that produce enzymes for the nitrification of ammonia and 16S rRNA. Furthermore, this study employed generalized linear models using lme4 in R to estimate the effects of the fertilizers, fungicides and growth regulators on the gene copy numbers of the target genes. 

## Experimental design

### Field trial of winter-wheat in a randomized complete block design



An agricultural field in experimental station ZALF in Brandenburg, Germany (53°21'58.5"N 13°48'13.3"E) was used to study the effects of agriculture on soil microorganisms. The site has been used agriculturally for more than 30 years.  On the field, 3x7m plots were arranged in blocks. On 18.09.2019 winter wheat (*Triticum aestivum*, variety: RGT Reform) was sown in a density of 280 seeds/$\mathrm{m^2}$ on the assigned plots.  To avoid cross-over effects a buffer of rye (*Secale cereal*) was sown between blocks.
```{r soilParameters, echo=FALSE, message=FALSE}

kbl(soilparam, booktabs = T, caption = "Parameters and concentrations of chemical compounds in the experimental sites soil. Measured by a previous study using the same site.")%>% #print table 
  kable_styling(position = "center")#%>%
  #footnote(general= table_note, 
           #general_title = "Note.", footnote_as_chunk = T, threeparttable = T) #enforces linebreak for footnote
```
The chemical and physical quality of the fields soil was determined by a previous study (Table \@ref(tab:soilParameters)), which used the same field. Nitrogen and magnesium were measured by $\mathrm{CaCl_2}$ extraction. Phosphorus, potassium and sodium was measured by the double lactate method. Calcium was measured by the calcium-acetate-lactate method. pH was measured in a KCl suspension. Total carbon, nitrogen and sulfur were measured in an elemental analyzer.  The topsoil on the site is a loamy sand and ranges from Su3 to Sl3 across the blocks.

### Treatment of fertilizer, fungicides and growth regulators 
All plots were treated with an herbicide in form of *Viper™ Compact (Corteva Agriscience)* to suppress weeds at 26 days after sowing,. The herbicide was applied to lower nutritional and spatial struggle between the wheat crop and plants of no interest. 
Treatment was categorized into two factors. The components were nitrogen fertilizer, sulfur fertilizer, magnesium fertilizer, fungicides and growth regulators. The fertilizers were grouped into one treatment factor. Fungicide and growth regulators were grouped into a second treatment factor. Both factors were binary (Application [Yes] / No Application [No]) and crossed. Resulting in one treatment for each factor plus one crossed treatment and a control treatment. Fertilizer treatment was applied as calcium ammonium nitrate (CAN), magnesium oxide (MgO) and sulfur (S). All were added to the wheat plots 179 days after sowing. To secure the nitrogen supply in the fertilized plots, another dosage of CAN was added 201 days after sowing. Fungicides were applied to the wheat plots as *Champion ® (BASF)* at 210 and *Seguris \copyright (Syngenta)* at 232 days after sowing. Growth regulators were applied in form of *Calma (NuFarm)* and *CCC 720 (Bayer Agrar)* both at 203 days after sowing. A detailed list of the herbicide that was applied to all plots, fertilizers, fungicides and growth regulators and their active ingredients is listed in Table \@ref(tab:Trt). 

```{r Trt, echo=FALSE, message=FALSE}
library(kableExtra)
trt %>%
   kbl(booktabs = T,
       escape = F, 
       caption = "Listing of all treatment components used in the experiments. Days of application and their active ingredients included. Only herbicide applied to all plots. Fertilizers were grouped into one treatment factor. Fungicides and growth regulator into the other.")%>%
   add_header_above(c(" " = 2, "Fertilizers" = 3, " " = 2))%>%
   add_header_above(c(" " = 2, "Factor 1" = 3, "Factor 2" = 2))%>%
   footnote(general = "DAS = Days after sowing")#print table 
  #footnote(general= table_note, 
           #general_title = "Note.", footnote_as_chunk = T, threeparttable = T) #enforces linebreak for footnote
```

The plots were sampled at the tillering (217 days after sowing) and stem elongation stage (265 days after sowing) of the wheat crop. Before the first sampling timepoint all treatments factors had been applied. Between the first and second sampling point another dosage of fungicide had been applied.
On five representative locations within each plot one crop was dug out with a spade. The crop was struck on the spade three times to remove loose bulk soil. The roots were then struck into a collection container to collect the rhizosphere soil. After collecting the rhizosphere soil sample, the root was separated from the shoot. Root and rhizosphere soil were stored separately in plastic bags. From the rhizosphere soil a representative subsample in a 15 ml falcon tube was taken.  All samples were stored on dry ice until the subsample of the rhizosphere soil could be stored at -80°C, root and the remaining rhizosphere soil at -20°C in the research center. The samples were kept in these conditions to prevent the microorganism community structure to change in the sample. 
In the laboratory the remaining soil and microorganisms attached to the roots were washed off. A subsample of the root was placed inside a 50 ml falcon tube. 1X Phosphate-buffered saline (PBS) solution (kept at 4°C) was added to detach and suspend the remaining soil and microorganisms attached to the roots. The tube was shaken by hand three times and shaken on a horizontal shaker for 10 min on 400 rpm. The washed roots were removed from the falcon tube. Again, the falcon tube was filled with 1x PBS to the 50 ml mark. The suspension was centrifuged at 4500 x g for 30 min at 4°C to pellet the washed off soil and microorganisms. After the centrifuging, the supernatant was discarded. The pellet was resuspended in 4 ml 1x PBS solution and vortexed. The suspension was transferred into 2 ml Eppendorf tubes and centrifuged at 4500 x g for 5 min. Again, the supernatant was discarded. The remaining liquid was stored in separate tubes at -80°C. Lastly the washed roots were dried at 60°C in a drying oven for three days and then milled.
The samples are named as the following in this thesis. The rhizosphere soil sampled at the field is named soil. The washed of soil from the root system is called rhizosphere (Table \@ref(tab:sample)). 

```{r sample, echo=FALSE, message=FALSE}
library(kableExtra)
smpl %>%
   kbl(booktabs = T,
       escape = F, 
       caption = "Sample sources")
  #footnote(general= table_note, 
           #general_title = "Note.", footnote_as_chunk = T, threeparttable = T) #enforces linebreak for footnote
```


## Chemical analysis soil and roots
Nitrogen concentration may vary in soil within treatments due to errors in application or varying spatial availability and influence microorganism community structures. Therefore, the plant available nitrate and ammonium concentration was measured in the plots. In addition, subsamples of the wet soils were dried at 20°C for 3 days, milled and analyzed for total carbon in nitrogen in an elemental analyzer. Same was done for the dried roots from the sampling process. 

## Amplification of target genes using PCR

The diet of soil-living microorganisms depends on their ability to alter chemical compounds. They use enzymes to catalyze favorable reaction of chemical compounds in and around the soil to gain energy for themselves. Their ability to produce these enzymes is dictated by the genome of the microorganisms. The blueprints for the enzymes are encoded in genes of the genome. The genes can be copied and multiplied by DNA polymerases. Here we use the DNA polymerases to copy genes of interest in the total genomic DNA of our samples and quantify them with DNA dyes.  
For this project genes of soil nitrifying microorganisms (transformation of ammonia to nitrate) were targeted. Ammonium-oxidizing archaea (AOA) and bacteria (AOB) are using ammonia monooxygenases (AMO) to oxidize ammonia in the soil. Under aerobic conditions ammonia is oxidized to hydroxylamine, which is then stepwise transformed to nitrate by other microorganisms. In contrast, Comammox (“complete ammonia oxidizers”) can completely oxidize ammonia to nitrate in one step and one bacterium. This process is used by the microorganisms to gain energy.
In amoA we targeted the subunit A in the sequence with established primers for ammonium-oxidizing archaea and bacteria (REFERENCE TABLE). For Comammox bacteria we used a primer 
set published in 2019 (Zhao et al. 2019). For general abundance we targeted the ribosomal 16s RNA.

```{r primerTable, echo=FALSE, message=FALSE}
library(kableExtra)
prmr %>%
   kbl(booktabs = T,
       escape = F, 
       caption = "Primer table", 
       col.names =linebreak(c("Target\ngene", "Kingdom", "Primer Sequence F (5-3')/\n Primer Sequence R (5-3')", "Subtarget"), align = "l"))
  #footnote(general= table_note, 
           #general_title = "Note.", footnote_as_chunk = T, threeparttable = T) #enforces linebreak for footnote
```