```{r include_packages_2, include = FALSE}
# This chunk ensures that the huskydown package is
# installed and loaded. This huskydown package includes
# the template files for the thesis and also two functions
# used for labeling and referencing
if(!require(devtools))
  install.packages("devtools", repos = "http://cran.rstudio.com")
if(!require(dplyr))
    install.packages("dplyr", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("ggplot2", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("bookdown", repos = "http://cran.rstudio.com")
if(!require(gauchodown)){
  library(devtools)
  devtools::install_github("benmarwick/gauchodown")
  }
library(gauchodown)
flights <- read.csv("data/flights.csv")
```


# Results {#ref-labels}

```{r chemly knitr child, echo=FALSE, error=FALSE,message=FALSE,results='asis'}
cat(knitr::knit_child("/sub/chemly.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

## Chemical analysis of soil and roots 


Soil samples of all treatment plots at the tillering and stem-elongation phase of winter-wheat crop were analyzed for their contents vital for crop and microorganisms growth. Nitrogen of wet soil samples were extracted by $\mathrm{KCl}$ from wet soil samples. Nitrate ($\mathrm{NO_3^-}$) and ammonia ($\mathrm{NH_4^+}$) were measured by ion-chromatography. 
Nitrate and ammonia concentrations are the highest (averaging at ~ 4 mg per 100g soil) in the fertilized plots at the first timepoint (Figure \@ref(fig:soilnitatmo)). Although both received the same amount of fertilizer (40 + 80 kg $\frac{CAN}{ha}$), the fully treated plots have half of the nitrate and ammonia concentration compared to the fertilized only plots. At the second sampling, the concentration dropped below 1 mg per 100g wet soil for both treatments on average. Concentration of nitrate decreased slightly more than ammonia. 
Without fertilization, the concentration of nitrate and ammonia was low and did not exceed 0.26 mg per 100g wet soil at both timepoints. At the first timepoint the nitrate concentration is higher while at the second timepoint the ammonium concentration is higher.  
Total carbon and nitrogen concentration was measured in air-dried soil samples at the first timepoint. Total carbon concentrations are in a similar range across all treatments (0.78-0.84%). Total nitrogen in the fertilized plots are 0.0966 % and 0.0918 % and in the unfertilized 0.0852 % and 0.0804 %. In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady. pH in the wet soils in all treatment plots are on average between 6.1 - 6.5 (data not shown).
In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady (\@ref(fig:rootctnt)). 

```{r soilnitatmo, echo=FALSE, fig.cap="Soil nitrogen content"}
soil
```


```{r rootctnt, echo=FALSE, fig.cap="dd content"}
root
```

## Gene copy numbers of target genes

16s rRNA and amoA of bacterial, archaea and commamox origin were amplified in quantitative polymerase chain reaction assays. As a template genomic DNA from the soil and rhizosphere of wheat crops was analyzed. Their abundance was estimated in gene copy numbers by comparing to a dilution series standard curve. The source of the standard was a synthetic gene fragment with a known amount of gene copy numbers. Two sample source at each two timepoints were evaluated in four assays per target gene. 

DNA templates tested in dilution to offset inhibition by co-extracted humic acids. The data was re-calculated as gene copy numbers per ng DNA of the targeted gene. The resulting raw datapoints are displayed in the upper swarmplots. At the right side of each group swarm plot two black bars represent the standard deviation and the gap between the two bars is the mean. The differences of means between treatment group and control is calculated, then resampled and repeated 5000 times. Around the results of the bootstrap re-sample a 95% confidence interval is created and displayed. The timepoints of each group are paired. Soil and rhizosphere sample sources are evaluated separately. 

### 16s rRNA

```{r 16s knitr child, echo=FALSE, error=FALSE,message=FALSE,results='asis'}
cat(knitr::knit_child("/sub/16ss.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

The V3 region of the 16s rRNA was amplified with/between 341F and 518R primers. The performance of the assay are similar. Intercept ranges between 33.7 and 36. Slopes are between -3.2 and -3.51 with a $\mathrm{R^{2}}$ $\geq$ 0.989. The Efficiency of amplification of the standard ranges from 92 - 102 %. The templates were diluted 1:400.  

```{r 16sStandardTable, echo=FALSE, message=FALSE}
library(kableExtra)
kbl(standard_16s, booktabs = T, caption = "16s Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", "R2", "Efficiency"))%>% #print table 
  kable_styling(position = "center")
```

In the soil samples, the raw data shows that 16s rRNA gene copy numbers are higher in the first timepoint in the fertilized (Fert+Full) plots, shortly after the fertilizer had been applied. In the control and only fungicide + growth regulator treated plots datapoints from the timepoints are mixed, but the second timepoint is more homogenely distributed. In the fungicide + growth regulator treated there are two extreme outliers. 

All treatments have a **positive** mean difference. Fungicides and growth regulator only treated plots has the highest mean difference. Except for the fully treated plots, the lower limit of the 95% CI is positive for all treatments.
```{r soil16s, echo=FALSE, fig.cap="Soil 16s rRNA estimation of mean differences"}
soil16s #estimation plots, paired mean differences, 5000 bootstrap ci 
```

In the rhizosphere samples the 16s rRNA gene copy numbers are  higher at the first timepoint on average. In two instances at the fertilized and one in the fully treated plots the gene copy numbers are higher in the second timepoint. The mean of differences of the fertilized is positive with 12300. Only the fully treated plots show an even higher mean differences of 46300. Fungicides and growth regulator only treated plots have a negative mean difference of -8890. But lower limits of the confidence intervals are negative for all treatments.

```{r rhizo16s, echo=FALSE, fig.cap="Rhizo 16s rRNA estimation of mean differences"}
rhizo16s #estimation plots, paired mean differences, 5000 bootstrap ci 
```

### Bacterial amoA

The subunit A of *amoA* of bacterial origin was targeted with 1F and 2R primers. 

\clearpage

<!-- clearpage ends the page, and also dumps out all floats.
  Floats are things like tables and figures. -->


