
```{r include_packages_2, include = FALSE}
# This chunk ensures that the huskydown package is
# installed and loaded. This huskydown package includes
# the template files for the thesis and also two functions
# used for labeling and referencing
if(!require(devtools))
  install.packages("devtools", repos = "http://cran.rstudio.com")
if(!require(dplyr))
    install.packages("dplyr", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("ggplot2", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("bookdown", repos = "http://cran.rstudio.com")
if(!require(gauchodown)){
  library(devtools)
  devtools::install_github("benmarwick/gauchodown")
  }
library(gauchodown)
flights <- read.csv("data/flights.csv")
```


# Results {#ref-labels}

```{r chemly knitr child, echo=FALSE, error=FALSE,message=FALSE,results='asis'}
cat(knitr::knit_child("/sub/chemly.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

## Chemical analysis of soil and roots 


Soil samples of all treatment plots at the tillering and stem-elongation phase of winter-wheat crop were analyzed for their contents vital for crop and microorganisms growth. Nitrogen of wet soil samples were extracted by $\mathrm{KCl}$ from wet soil samples. Nitrate ($\mathrm{NO_3^-}$) and ammonia ($\mathrm{NH_4^+}$) were measured by ion-chromatography. 
Nitrate and ammonia concentrations are the highest (averaging at ~ 4 mg per 100g soil) in the fertilized plots at the first timepoint (Figure \@ref(fig:soilnitatmo)). Although both received the same amount of fertilizer (40 + 80 kg $\frac{CAN}{ha}$), the fully treated plots have half of the nitrate and ammonia concentration compared to the fertilized only plots. At the second sampling, the concentration dropped below 1 mg per 100g wet soil for both treatments on average. Concentration of nitrate decreased slightly more than ammonia. 
Without fertilization, the concentration of nitrate and ammonia was low and did not exceed 0.26 mg per 100g wet soil at both timepoints. At the first timepoint the nitrate concentration is higher while at the second timepoint the ammonium concentration is higher.  
Total carbon and nitrogen concentration was measured in air-dried soil samples at the first timepoint. Total carbon concentrations are in a similar range across all treatments (0.78-0.84%). Total nitrogen in the fertilized plots are 0.0966 % and 0.0918 % and in the unfertilized 0.0852 % and 0.0804 %. In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady. pH in the wet soils in all treatment plots are on average between 6.1 - 6.5 (data not shown).
In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady (\@ref(fig:rootctnt)). 

```{r soilnitatmo, echo=FALSE, fig.cap="Soil nitrogen content"}
soil
```


```{r rootctnt, echo=FALSE, fig.cap="dd content"}
root
```

## Gene copy numbers of target genes

16s rRNA and amoA of bacterial, archaea and commamox origin were amplified in quantitative polymerase chain reaction assays. As a template genomic DNA from the soil and rhizosphere of wheat crops was analyzed. Their abundance was estimated in gene copy numbers by comparing to a dilution series standard curve. The source of the standard was a synthetic gene fragment with a known amount of gene copy numbers. Two sample source at each two timepoints were evaluated in four assays per target gene. 

DNA templates tested in dilution to offset inhibition by co-extracted humic acids. The data was re-calculated as gene copy numbers per ng DNA of the targeted gene. The resulting raw datapoints are displayed in the upper swarmplots. At the right side of each group swarm plot two black bars represent the standard deviation and the gap between the two bars is the mean. The differences of means between treatment group and control is calculated, then resampled and repeated 5000 times. Around the results of the bootstrap re-sample a 95% confidence interval is created and displayed. Soil and rhizosphere sample sources are evaluated separately. 



### 16s rRNA
<!-- Ideas
* Merge estimation plots soil and rhizo into one with ggarrange (plots are ggplot2s)
-->
```{r 16s knitr child, echo=FALSE, error=FALSE,message=FALSE,results='asis'}
cat(knitr::knit_child("/sub/16ss.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

The V3 region of the 16s rRNA was amplified with/between 341F and 518R primers. The performance of the assay are similar. Intercept ranges between 33.7 and 36. Slopes are between -3.2 and -3.51 with a $\mathrm{R^{2}}$ $\geq$ 0.989. The Efficiency of amplification of the standard ranges from 92 - 102 % (Table \@ref(tab:16sStandardTable). The templates were diluted 1:400.  

```{r 16sStandardTable, echo=FALSE, message=FALSE}
library(kableExtra)
kbl(standard_16s, booktabs = T, caption = "16s Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", "R2", "Efficiency"))%>% #print table 
  kable_styling(position = "center")
```

In the soil samples, the raw data shows that 16s rRNA gene copy numbers are higher in the first timepoint in the fertilized (Fert+Full) plots, shortly after the fertilizer had been applied. In the control and only fungicide + growth regulator treated plots datapoints from the timepoints are mixed, but the second timepoint is more homogenely distributed. In the fungicide + growth regulator treated there are two extreme outliers. 

All treatments have a **positive** mean difference. Fungicides and growth regulator only treated plots has the highest mean difference. Except for the fully treated plots, the lower limit of the 95% CI is positive for all treatments (Figure \@ref(fig:soil16s)). The numerical difference between two ticks on the y-axis is for the upper plot 250.000 and on the lower is a 100.000 gene copy numbers. 

```{r soil16s, echo=FALSE, fig.cap="Soil 16s rRNA estimation of mean differences"}
soil16s #estimation plots, paired mean differences, 5000 bootstrap ci 
```

In the rhizosphere samples the 16s rRNA gene copy numbers are  higher at the first timepoint on average. In two instances at the fertilized and one in the fully treated plots the gene copy numbers are higher in the second timepoint. The mean of differences of the fertilized is positive with 12300. Only the fully treated plots show an even higher mean differences of 46300. Fungicides and growth regulator only treated plots have a negative mean difference of -8890. But lower limits of the confidence intervals are negative for all treatments (Figure \@ref(fig:rhizo16s)).



```{r rhizo16s, echo=FALSE, fig.cap="Rhizo 16s rRNA estimation of mean differences"}
rhizo16s
```

\clearpage

<!-- clearpage ends the page, and also dumps out all floats.
  Floats are things like tables and figures. -->
  
```{r aob knitr child, include=FALSE, eval=TRUE}
cat(knitr::knit_child("/sub/aob.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

### Bacterial amoA

The subunit A of *amoA* of bacterial origin was targeted with 1F and 2R primers. The performance of the assays with the rhizosphere samples showed poor performance (\@ref(tab:aobtable)) of the standard amplification. The high slope value of > -2.4 resulted in a high efficiency > 160% and low intercept < 30. For the soil samples assays the performance of the standard amplification is good with slope value approximating -3.3 and a intercept close to 37. $\mathrm{R^{2}}$ for the rhizosphere samples are > 0.98 and for the soil samples > 0.99 (Table \@ref(tab:aobStandardTable). Templates were diluted 1:20

The poor standard curve performance in the rhizosphere sample assays resulted in a biased estimation of their gene copy numbers. Therefore, their $\mathrm{C_Q}$ values were fitted on another standard curve. All rhizosphere samples and the soil samples at the first timepoint were fitted on the standard curve of the soil samples at the second timepoint and gene copy numbers calculated. 

```{r aobStandardTable, echo=FALSE, message=FALSE}
kbl(standard_aob, booktabs = T, caption = "AOB Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", "R2", "Efficiency"))%>% #print table 
  kable_styling(position = "center")%>%
  footnote(general = "All assays were calibrated on the assays standard curveof the soil sample at second timepoint", 
           threeparttable = TRUE)
```

In addition to poor performance of standard amplification , unspecific amplification in the rhizosphere templates was detected. Generally, unspecific amplification formed in all assays in the negative controls as primer dimers. In the rhizosphere samples the primer dimers were also detected in the templates.
Figure \@ref(fig:AobMelt) shows the melt curve of rhizosphere samples from the second timepoint. It shows a peak at 75°C and 89°C. The first peak are primer dimers, while the latter is the *amoA* . The 75°C peaks are in some cases larger than the same peaks in the negative control. The negative controls are the blue-lines peaking at 75°C with no peak at 89°C. 


```{r AobMelt, echo=FALSE, fig.cap="Assays melt curve of rhizosphere at second timepoint"}
library(knitr)
include_graphics("melt/aobrhizot3melt.jpg")
```

The bacteria community in the soil samples have higher gene copy numbers of *amoA* at the second timepoint on average. At the first timepoint the raw values  are close to zero. Compared to the control, all treatments produced a higher mean of differences in gene copy numbers of *amoA* (Figure \@ref(fig:soilAob)). The fully treated plots show the highest mean of difference with 644 and a positive lower confidence interval limit compared to the control. The fertilized plots have 406 mean of differences in gene copy numbers wand the fungicides and growth regulator treated plots a 276 mean of differences. Both have a negative lower confidence interval limit. 

```{r soilAob, echo=FALSE, fig.cap="Soil amoA bacteria estimation"}
soilaob
```

In the direct rhizosphere of the wheat crops there are lower gene copy numbers of *amoA* at the second timepoint for the control, fertilized and fungicides + growth regulator treated plots. The raw values in the fully treated plots are heterogenly distributed between timepoints. The fully treated plots show the highest mean of differences with 306 gene copy numbers, while the only fertilized has even a negative value of -377. With fungicides and growth regulator treated plots produced a slightly positive mean of difference with 20.3. All lower limits for the confidence intervals are negative. 

```{r rhizoAob, echo=FALSE, fig.cap="Rhizo amoA bacteria estimation"}
rhizoaob
```

\clearpage

### Archaea *amoA*

```{r aoa knitr child, include=FALSE, eval=TRUE}
cat(knitr::knit_child("/sub/aoa.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

Subunit A of *amoA* in Archaea was targeted with CrenamoA23F and CrenamoA616R primers in the samples. The slopes of the standard curve (Table \@ref(tab:aoaStandardTable)) are ranging between -3.8 and -4.6 with intercepts between 41 - 44. This resulted in a low efficiency of the standard amplification (66 - 83 %). $\mathrm{C_Q}$ of the technical replicates of the standard are conistent which reflects in $\mathrm{R^{2}}$ values > 0.99. \par


```{r aoaStandardTable, echo=FALSE, message=FALSE}
kbl(standard_aoa, booktabs = T, caption = "AOA Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", "R2", "Efficiency"))%>% #print table 
  kable_styling(position = "center")
 
```

In the soil samples, *amoA* gene copy numbers are in the ranges of 10,000 - 40,000 (Figure \@ref(fig:soilAoa)). In the control plots, gene copy numbers are higher at the second timepoint than at the first. No clear pattern between timepoints is observable for the treated plots. The mean of differences of the fully treated plots is the most negative mean of differences (-8310). The upper and lower limits of the fully treated plots are negative. The fertilized and the fungicide + growth regulator treated plots have also negative lower CI limits, but a positive upper limit. Their mean of differences is also negative with -3500 for the fertilized and -1010 for the fungicide and growth regulator treated plots. 

```{r soilAoa, echo=FALSE, fig.cap="Rhizo amoA archaea estimation"}
soilaoa
```

The gene copy numbers of archaea *amoA* in the rhizosphere samples have a similar range of 10,000 - 30,000 to the soil samples. One outlier above 50,000 in the fully treated plots is observable. The second timepoint has on average a higher gene copy numbers of *amoA* than the first. The mean of differences is the most negative in the fungicide + growth regulator treated plots (-2800). The fully treated plots have a mean of differences of -1040. Only the fertilized plots show a positive mean of differences at 7.65. All treatments have a negative lower limit and a positive upper limit. The high outlier of the fully treated plot caused a high upper limit of their confidence interval. 

```{r rhizoAoa, echo=FALSE, fig.cap="Rhizo amoA archaea estimation"}
rhizoaoa
```


### Complete ammonia oxidiser (Comammox)

```{r com knitr child, include=FALSE, eval=TRUE}
cat(knitr::knit_child("/sub/com.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

A conserved region in the genome shared by complete ammonia oxidiser bacteria was target with XY primers.


```{r comStandardTable, echo=FALSE, message=FALSE}
kbl(standard_com, booktabs = T, caption = "AOA Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", "R2", "Efficiency"))%>% #print table 
  kable_styling(position = "center")
 
```
