---
# UCSB thesis fields
title: "Improving the Management of Marine Resources through Economics and Data Science"
author: "Daniel A. Ovando"
degree_year: "2018"
degree_month: "June"
defense_year: "2018"
defense_month: "May"
program: "Slowly and Painfully Working Out the Surprisingly Obvious"
uc_campus: "Santa Barbara"
report: "Dissertation"
degree: "Doctor of Philosphy"
chair: "Professor Christopher Costello"
signature1: "Professor Steven Gaines"
signature2: "Professor Ray Hilborn"
signature3: "Professor Olivier Deschenes"
abstract: "The data say 'meh'"
acknowledgments: "Thanks everyone!"
dedication: "To Hobbes"
# End of UCSB thesis fields
knit: "bookdown::render_book"
site: bookdown::bookdown_site
output: 
  gauchodown::thesis_pdf:
    latex_engine: xelatex
  # gauchodown::thesis_gitbook: default
  # gauchodown::thesis_word: default
  # gauchodown::thesis_epub: default
bibliography: bib/thesis.bib
# Download your specific bibliography database file and refer to it in the line above.
csl: csl/ieee-with-url.csl
# Download your specific csl file and refer to it in the line above.
link-citations: yes
linkcolor: blue
citecolor: blue
urlcolor: blue
lot: true # create list of tables
lof: true # create list of figures
#space_between_paragraphs: true
# Delete the # at the beginning of the previous line if you'd like
# to have a blank new line between each paragraph
header-includes:
- \usepackage{mhchem}
---

# UCSB thesis fields

Placeholder



<!--chapter:end:index.Rmd-->

<!-- The {.unnumbered} option here means that the introduction will be "Chapter 0." You can also use {-} for no numbers on chapters. -->

# Introduction {#intro}

```{r chunk1}
getwd()
```

test 

Welcome to the _R Markdown_ thesis template. This template is based on (and in many places copied directly from) the UW LaTeX template, but hopefully it will provide a nicer interface for those that have never used TeX or LaTeX before. Using _R Markdown_ will also allow you to easily keep track of your analyses in **R** chunks of code, with the resulting plots and output included as well. The hope is this _R Markdown_ template gets you in the habit of doing reproducible research, which benefits you long-term as a researcher, but also will greatly help anyone that is trying to reproduce or build onto your results down the road.

Hopefully, you won't have much of a learning period to go through and you will reap the benefits of a nicely formatted thesis. The use of LaTeX in combination with _Markdown_ is more consistent than the output of a word processor, much less prone to corruption or crashing, and the resulting file is smaller than a Word file. While you may have never had problems using Word in the past, your thesis is likely going to be at least twice as large and complex as anything you've written before, taxing Word's capabilities. After working with _Markdown_ and **R** together for a few weeks, we are confident this will be your reporting style of choice going forward.

<!--  If you're still on the fence about using _R Markdown_, check out the resource for newbies available at <https://ismayc.github.io/rbasics-book/> .  -->

**Why use it?**

_R Markdown_ creates a simple and straightforward way to interface with the beauty of LaTeX.  Packages have been written in **R** to work directly with LaTeX to produce nicely formatting tables and paragraphs. In addition to creating a user friendly interface to LaTeX, _R Markdown_ also allows you to read in your data, to analyze it and to visualize it using **R** functions, and also to provide the documentation and commentary on the results of your project.  Further, it allows for **R** results to be passed inline to the commentary of your results.  You'll see more on this later.  

<!--  Having your code and commentary all together in one place has a plethora of benefits! -->

**Who should use it?**

Anyone who needs to use data analysis, math, tables, a lot of figures, complex cross-references, or who just cares about the final appearance of their document should use _R Markdown_. Of particular use should be anyone in the sciences, but the user-friendly nature of _Markdown_ and its ability to keep track of and easily include figures, automatically generate a table of contents, index, references, table of figures, etc. should make it of great benefit to nearly anyone writing a thesis project.

<!--chapter:end:00-intro.Rmd-->


# R Markdown Basics {#rmd-basics}

Placeholder


## Lists
## Line breaks
## R chunks
## Inline code
## Including plots
## Loading and exploring data
## Additional resources

<!--chapter:end:01-chap1.Rmd-->


# Mathematics and Science {#math-sci}

Placeholder


## Math
## Chemistry 101: Symbols
### Typesetting reactions
### Other examples of reactions
## Physics
## Biology

<!--chapter:end:02-chap2.Rmd-->

```{r include_packages_2, include = FALSE}
# This chunk ensures that the huskydown package is
# installed and loaded. This huskydown package includes
# the template files for the thesis and also two functions
# used for labeling and referencing
if(!require(devtools))
  install.packages("devtools", repos = "http://cran.rstudio.com")
if(!require(dplyr))
    install.packages("dplyr", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("ggplot2", repos = "http://cran.rstudio.com")
if(!require(ggplot2))
    install.packages("bookdown", repos = "http://cran.rstudio.com")
if(!require(gauchodown)){
  library(devtools)
  devtools::install_github("benmarwick/gauchodown")
  }
library(gauchodown)
flights <- read.csv("data/flights.csv")
```


# Results {#ref-labels}

```{r chemly, echo=FALSE, error=FALSE,message=FALSE,results='asis'}
cat(knitr::knit_child("/sub/chemly.Rmd", quiet=TRUE))
#knits the calculations for chemical analysis
```

## Chemical analysis of soil and roots 


Soil samples of all treatment plots at the tillering and stem-elongation phase of winter-wheat crop were analyzed for their contents vital for crop and microorganisms growth. Nitrogen of wet soil samples were extracted by $\mathrm{KCl}$ from wet soil samples. Nitrate ($\mathrm{NO_3^-}$) and ammonia ($\mathrm{NH_4^+}$) were measured by ion-chromatography. 
Nitrate and ammonia concentrations are the highest (averaging at ~ 4 mg per 100g soil) in the fertilized plots at the first timepoint (Figure \@ref(fig:soilnitatmo)). Although both received the same amount of fertilizer (40 + 80 kg $\frac{CAN}{ha}$), the fully treated plots have half of the nitrate and ammonia concentration compared to the fertilized only plots. At the second sampling, the concentration dropped below 1 mg per 100g wet soil for both treatments on average. Concentration of nitrate decreased slightly more than ammonia. 
Without fertilization, the concentration of nitrate and ammonia was low and did not exceed 0.26 mg per 100g wet soil at both timepoints. At the first timepoint the nitrate concentration is higher while at the second timepoint the ammonium concentration is higher.  
Total carbon and nitrogen concentration was measured in air-dried soil samples at the first timepoint. Total carbon concentrations are in a similar range across all treatments (0.78-0.84%). Total nitrogen in the fertilized plots are 0.0966 % and 0.0918 % and in the unfertilized 0.0852 % and 0.0804 %. In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady. pH in the wet soils in all treatment plots are on average between 6.1 - 6.5 (data not shown).
In the dried roots of the unfertilized wheat crops the total carbon and nitrogen decreased slightly. In the fertilized crops it increased slightly or stayed steady (\@ref(fig:rootctnt)). 

```{r soilnitatmo, echo=FALSE, fig.cap="Soil nitrogen content"}
soil
```


```{r rootctnt, echo=FALSE, fig.cap="dd content"}
root
```

## Gene copy numbers of target genes

16s rRNA and amoA of bacterial, archaea and commamox origin were amplified in quantitative polymerase chain reaction assays. As a template genomic DNA from the soil and rhizosphere of wheat crops was analyzed. Their was estimated in gene copy numbers by comparing to dilution series standard curve of a synthetic standard with a known amount of gene copy numbers. 

### 16s rRNA

The V3 region of the 16s rRNA was amplified with/between 341F and 518R primers. 

```{r 16sStandard, echo=FALSE}

standard_16s <- #load data
      read_delim("data/16s_standard.csv",",",)

kbl(standard_16s, booktabs = T, caption = "16s Standard",
    col.names = c("Source", "Timepoint", "Slope", "Intercept", expression("R"^"2"), "Efficiency"))%>% #print table 
  kable_styling(position = "center")
```

\clearpage

<!-- clearpage ends the page, and also dumps out all floats.
  Floats are things like tables and figures. -->



<!--chapter:end:03-chap3.Rmd-->

# Conclusion {-}

If we don't want Conclusion to have a chapter number next to it, we can add the `{-}` attribute.

**More info**

And here's some other random info: the first paragraph after a chapter title or section head _shouldn't be_ indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there.


<!--chapter:end:04-conclusion.Rmd-->

`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

<!--
If you feel it necessary to include an appendix, it goes here.
-->


# The First Appendix

This first appendix includes all of the R chunks of code that were hidden throughout the document (using the `include = FALSE` chunk tag) to help with readibility and/or setup.

**In the main Rmd file**

```{r ref.label='include_packages', results='hide', echo = TRUE}
```

**In Chapter \@ref(ref-labels):**

```{r ref.label='include_packages_2', results='hide', echo = TRUE}
```

# The Second Appendix, for Fun

<!--chapter:end:05-appendix.Rmd-->

# Colophon {-}

This document is set in [EB Garamond](https://github.com/georgd/EB-Garamond), [Source Code Pro](https://github.com/adobe-fonts/source-code-pro/) and [Lato](http://www.latofonts.com/lato-free-fonts/). The body text is set at 11pt with $\familydefault$. 

It was written in R Markdown and $\LaTeX$, and rendered into PDF using [gauchodown](https://github.com/danovando/gauchodown) and [bookdown](https://github.com/rstudio/bookdown). 

This document was typeset using the XeTeX typesetting system, and the [University of Washington Thesis class](http://staff.washington.edu/fox/tex/) class created by Jim Fox. Under the hood, the [University of Washington Thesis LaTeX template](https://github.com/UWIT-IAM/UWThesis) is used to ensure that documents conform precisely to submission standards. Other elements of the document formatting source code have been taken from the [Latex, Knitr, and RMarkdown templates for UC Berkeley's graduate thesis](https://github.com/stevenpollack/ucbthesis), and [Dissertate: a LaTeX dissertation template to support the production and typesetting of a PhD dissertation at Harvard, Princeton, and NYU](https://github.com/suchow/Dissertate)

The source files for this thesis, along with all the data files, have been organised into an R package, xxx, which is available at https://github.com/xxx/xxx. A hard copy of the thesis can be found in the University of Washington library.

This version of the thesis was generated on `r Sys.time()`. The repository is currently at this commit:

```{r echo = FALSE, eval = FALSE}
# I've set eval=FALSE to ensure Travis-CI can run
# if you're not using Travis-CI, then eval=TRUE will be fine
# library(git2r)
# if ( git2r::in_repository() ) {
#        summary(commits()[[1]]) 
# } else { 
#        message("We are not in a git repository") 
# }
```

The computational environment that was used to generate this version is as follows:

```{r echo = FALSE}
devtools::session_info()
```


<!--chapter:end:98-colophon.Rmd-->


# References {-}

Placeholder



<!--chapter:end:99-references.Rmd-->

